src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -391,6 +401,75 @@ protected virtual void Initialize()
             }
         }
 
+        // Scan the extensions directory and Load custom extension.
+        // $$$ - need to finalize this policy. Currently enabled via an AppSettting. 
+        private void LoadCustomExtensions(JObject hostConfig)
+        {
+            var bindingRoot = ConfigurationManager.AppSettings["BYOB_Path"];
```

Comment:
Also, I think you should add this as a config property on ScriptHostConfiguration, e.g. ExtensionsPath, as I had done in my prototype https://github.com/Azure/azure-webjobs-sdk-script/compare/extensibility